### PR TITLE
Python#97

### PR DESCRIPTION
**Pull request by bathomas**
Working on #97.

### CODE REVIEW

**Comment by KrisThielemans** [comment]
now fails with
```
/Users/travis/.travis/job_stages: line 57: /usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/bin/python3.6: No such file or directory
```
clearly, the following in .travis.yml are unsafe
```
        export PYINST=/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions
        export PY_EXE=$PYINST/$PYMVER.6/bin/python3.6
```

**Comment by rijobro** [comment]
from the log:
`==> Upgrading 1 outdated package, with result: python 3.6.4_3`
So its looking for 3.6.4_2 but should be looking for 3.6.4_3. So yes, it looks dangerous to hardwire the python path.

File: .travis.yml [hunk 1]
```diff
@@ -127,8 +127,15 @@ before_install:
         export BUILD_FLAGS="$BUILD_FLAGS -DPYTHON_LIBRARY=$PYINST/2.7/lib/libpython2.7.dylib"
         export BUILD_FLAGS="$BUILD_FLAGS -DPYTHON_INCLUDE_DIR=$PYINST/2.7/include/python2.7"
       else
+        brew upgrade python || travis_terminate 1
         brew install python3 || travis_terminate 1
-        export PYINST=/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions
+
+        TARGETPYVER=`find /usr/local/Cellar/python3/* -type d  -maxdepth 0 | sed 's#.*/##' | sort -V | tail -n 1`
```
Comment (KrisThielemans) [inline .travis.yml]: ok. better. but you cannot have the *. find needs a single argument for the directory. just delete it.

**Comment by KrisThielemans** [comment]
I've fixed this  in #118 in a related manner, but simpler for the libraries. I didn't use the `find` trick which might come in handy later.

**Pull request closed without merging**